src/typeDefs/queries.ts

```diff
@@ -107,7 +107,7 @@ export const queries = gql`
       first: Int
       skip: Int
       orderBy: PostOrderByInput
-    ): PostConnection
+    ): PostsConnection
```

Comment:
this should be exposed as `Organization.posts`

Reply:
sorry sir but I haven;t created this ,this was already there

Reply:
Though it's not your code, the functionality will be affected. Please fix

Reply:
so I have to change this functionality to return simple post object ?

Reply:
@AVtheking It would break these instances in talawa-admin:-
https://github.com/search?type=code&auto_enroll=true&q=repo%3APalisadoesFoundation%2Ftalawa-admin+postsByOrganizationConnection

Reply:
ideally best practices and designs should be followed from the start so major disruptions aren't caused in the future, but right now talawa applications are not in that state

Right now the practice should be:-

add new functionalities without breaking the existing ones -> migrate the clients to use the new functionalities -> deprecate/remove the old functionalities

Reply:
> so I have to change this functionality to return simple post object ?

It's not a simple post object, it's just named incorrectly `PostConnection` instead of `PostsConnection`, like I had said `postsByOrganizationConnection` should be replaced with `Organization.posts` if possible, if you're up for the task make an issue in talawa-admin and upon merging of this PR fix the issues caused by the changes in this PR.

Reply:
so sir  I have to change to change it to Organization.posts and fix that in admin also ?

Reply:
![image](https://github.com/PalisadoesFoundation/talawa-api/assets/132201033/d2c632b0-d8df-430a-aba0-765202da019d)
this query is also getting the posts of the organization

Reply:
> so sir I have to change to change it to Organization.posts and fix that in admin also ?

yes, if you intend to remove the old field and make talawa-admin use the new field

Reply:
Though it's not necessary that you fix it, you could also open issues on those repositories and corresponding slack channels and inform the contributers about the breaking changes

Reply:
![image](https://github.com/PalisadoesFoundation/talawa-api/assets/132201033/699baac2-2ce7-4fd2-954a-411fa55a0d58)
so I have to remove that query and paste it here in the organization type as posts and move the implementation of the query inside the organization ?

Do I also need to change the pagination logic of this function ?

Reply:
@AVtheking just like you have done for `User.posts` you would do for `Organization.posts`, the difference would be that instead of filtering the posts by user_id(parent._id) you would be filtering with organization_id(parent._id).

Reply:
Wouldn't it requires where as argument sir to filter the posts ?

Reply:
@xoldyckk I have removed those queries and made it in the organization,If it needs to include where as argument also then let me now I will change it .

Reply:
@AVtheking filtering and sorting are expensive operations and therefore should only be added when client apps need it, the fields being exposed must also be considered, not all fields are indexed in database